test/test_isogram.c

```diff
@@ -1,36 +1,20 @@
 
```

Comment:
i think this found its way into the wrong file?
can you move this to iscogram.c